nucypher/config/constants.py

```diff
@@ -1,3 +1,23 @@
-DEFAULT_CONFIG_FILE_LOCATION = "cool winnebago"
-DEFAULT_CONFIG_ROOT = "my uncle has one just like it."
-DEFAULT_REST_PORT = 9151
\ No newline at end of file
+import os
+from os.path import abspath, dirname
+
+from appdirs import AppDirs
+
+import nucypher
+
+# Base Filepaths
+BASE_DIR = abspath(dirname(dirname(nucypher.__file__)))
+PROJECT_ROOT = abspath(dirname(nucypher.__file__))
+APP_DIR = AppDirs("nucypher", "NuCypher")
+
+# Configuration File
+TEMPLATE_CONFIG_FILE_LOCATION = os.path.join(BASE_DIR, 'cli', '.nucypher.ini')
```

Comment:
We should keep in mind that the `.ini` file will unlikely be alongside the code if `nucypher` is installed with `setup.py` into system directories. Ok for now, but we'll come across this once we start testing with properly installed `nucypher` package

Reply:
Perhaps, easy enough to fix even in this PR: just need to include `.nucypher.ini` in `package_data` in `setup.py`. We already have Solidity files there

Reply:
I'm using this hidden file as a 'template' to write new configuration file via the CLI (perhaps interactively at some point) and a method: `NodeConfiguration.write_default_config_file(filepath=/my/config/file/path/)`.  If I understand correctly, if we ship a required default config file then we'll need to install it with `setup.py`, but for now it's optional.